site/app/views/grading/ElectronicGraderView.php

```diff
@@ -66,6 +66,12 @@ public function statusPage(
         $peer_total = 0;
         $peer_graded = 0;
         $peer_percentage = 0;
+        $entire_peer_graded = 0;
+        $entire_peer_total = 0;
+        $total_students_submitted = 0;
```

Comment:
total_students_submitted is an input for the statusPage method. I don't think it should be immediately overrided with 0?

Reply:
Yes, I thought it was anyway to be set when $peer and $peerGradeable were both set (Non Peer View for Peer Gradeable), but it is extraneous here, Removed it.